views/templates.pug

```diff
@@ -230,8 +221,18 @@
             button.btn.btn-light.btn-sm.wrap-lines(type="button" title="Wrap lines" data-bind="wrap" aria-pressed="false" aria-label="Wrap lines")
               span Wrap lines
             input.d-none(type="checkbox" checked=false)
-          input.options.form-control(type="text" placeholder="Tool options..." size="256" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false")
+        button.btn.btn-light.btn-sm.toggle-args
+          span.fas.fa-terminal
+          span.hideable &nbsp;Arguments
+        button.btn.btn-light.btn-sm.toggle-stdin
+          span.fas.fa-sign-in-alt
+          span.hideable &nbsp;Stdin
+    .top-bar.btn-toolbar.bg-light.panel-args.d-none(role="toolbar")
+      input.options.form-control(type="text" placeholder="Tool arguments..." size="256" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false")
+    .top-bar.btn-toolbar.bg-light.panel-stdin.d-none(role="toolbar")
+      textarea.tool-stdin.form-control(placeholder="Tool stdin..." cols="1024" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" aria-multiline="false" style="resize: none")
```

Comment:
Note that, at least on firefox, this is still resizable :/

Reply:
ahhhhh so this might be why I couldn't get the css to work without !important? it is meant to be resizable

Reply:
Most likely yes! Note though that resizing stdin breaks the UI as there is no resize event fired. (And none is ever triggered in fact)

Reply:
Hadn't noticed the UI breaking, though I think it's acceptable in this case as long as there are no alternatives

Reply:
There are some interesting ways to get it to generate an event posted in this SO thread https://stackoverflow.com/questions/5570390/resize-event-for-textarea

The MutationObserver seems to be supported by Chrome, Firefox, IE11 and Edge

Reply:
I fixed the template so that it says there its resizable instead of having it in the css. Also added this mutationobserver event. Though I noticed eventhough IE and Edge have those classes, they don't actually offer the resize functionality to textareas. So only Chrome and Firefox (and presumably Safari and Chrome-Edge) users will be able to resize their textarea.